src/main/java/org/opentest4j/MultipleFailuresError.java

```diff
@@ -34,15 +36,15 @@
 
 	private static final long serialVersionUID = 1L;
 
-	private final List<AssertionError> failures = new ArrayList<AssertionError>();
+	private final List<Throwable> failures = new ArrayList<Throwable>();
 
 	private final String heading;
 
 	public MultipleFailuresError(String heading) {
 		this.heading = isBlank(heading) ? "Multiple Failures" : heading.trim();
 	}
 
-	public void addFailure(AssertionError failure) {
+	public void addFailure(Throwable failure) {
```

Comment:
@marcphilipp I certainly can do the change it his PR. But in my opinion I think this warrants a separate issue because this changes the contract of this class and changes how it can be used. I would have liked we have a separate issue that clearly states why we do it. My change also changed the contract but because throwable was a superclass it had minimal impact.

Please let me know what you prefer.